data/core.yaml

```diff
@@ -260,6 +260,7 @@ en:
     save: Upload
     cancel: Cancel
     changes: "{count} Changes"
+    download_changes: Download OsmChange file
```

Comment:
The standard spelling for OSC files is "osmChange"

Reply:
Thanks, I'll change it.  I was going off of the [osm wiki](https://wiki.openstreetmap.org/wiki/OsmChange) - should we change it there?

Reply:
Wiki page titles have to begin with a capital character.